### PR TITLE
revise README

### PR DESCRIPTION
**Pull request by r10s**
- remove some TODO and vague things (we should not really write things like that, it is not that someone picks that TODO up at some point :)
- move advanved things that are most times not needed to a separate section to make things look as easy as they are
- adapt overall style and case a bit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,4 +68,3 @@ each emulated peer will get the same `localStorage`.
 to really test the storage usage of your Webxdc,
```
Comment (adbenitez) [inline README.md:68]: this last note should probably point to the webxdc-dev tool, that is a better emulator supporting per tab/device localStorage

File: README.md [hunk 2]
```diff
@@ -2,21 +2,24 @@
...
 
 1. Open `index.html` in your web browser
 2. Click 'Add Peer' to open as many peers as you like
 3. Type a message and press 'Send' to see the update in each peer. (For Safari you might need to check the setting under Develop > Disable Local File Restrictions.)
 
+
 ## Developing webxdc apps
 
-Simply copy `webxdc.js` from this repo beside your `index.html` and you are ready to go.
+Simply copy `webxdc.js` from this repo beside your `index.html` and you are ready to go
```
Comment (adbenitez) [inline README.md:15]: I think this repo should be converted in a template, so it is easy to start a new github project clicking a button and having this project structure and files already in place.

We already have some templates, check:
https://github.com/orgs/webxdc/repositories?q=&type=template&language=&sort=

Converting to template is easy, go to https://github.com/webxdc/hello/settings and mark the first checkbox "Template repository"

File: README.md (hunk 1 above)
Comment (r10s) [reply README.md]: i added a line.

**Review by adbenitez — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (r10s) [reply README.md:15]: ah, interesting!

Comment (r10s) [reply README.md:15]: will do 👊

**Pull request merged**